src/uwtools/tests/test_rocoto.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable=missing-function-docstring
+"""
+Tests for uwtools.rocoto module.
+"""
+from importlib import resources
+
+import yaml
+
+from uwtools import rocoto
+from uwtools.tests import support
+
+# Test functions
+
+
+def test_add_jobname():
+    expected = yaml.safe_load(
+        """
+task_hello:
+  command: echo hello 
+  jobname: hello
+metatask_howdy:
+  var:
+    mem: 1 2 3
+  task_howdy_#mem#:
+    command: echo hello 
+    jobname: howdy_#mem#
+  task_hola_#mem#:
+    command: echo hello 
+    jobname: hola_#mem#
+  metatask_hey_#mem#:
+    var:
+      day: mon tues
+    task_hey_#mem#_#day#:
+      command: echo hello 
+      jobname: hey_#mem#_#day#
+"""
+    )
+
+    tasks = yaml.safe_load(
+        """
+task_hello:
+  command: echo hello
+metatask_howdy:
+  var:
+    mem: 1 2 3
+  task_howdy_#mem#:
+    command: echo hello 
+  task_hola_#mem#:
+    command: echo hello
+  metatask_hey_#mem#:
+    var:
+      day: mon tues
+    task_hey_#mem#_#day#:
+      command: echo hello
+"""
+    )
+
+    # pylint: disable=protected-access
```

Comment:
That's the right fix, but since accessing protected (private) members is commonplace in unit tests, this should go at the top of the test module so that it applied throughout, to avoid needing to repeat it. You'll see that a lot of other test modules have this `pylint` annotation.